Global style

willmt80 opened the pull request:
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/views/12631042/pulses/829576071)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

This allows developers to use antd components like text and buttons with the automatic styles that are used throughout the application

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

All of the variables are located within the craco.config.js file. The colors are at the top of the page and they correspond to different greens and greys that exist in the figma designs. Any of these values can be overridden in less files, see the info-card.less file for an example of overriding antd global styles.

## Screenshots

Provide screenshots of any new components, styling changes, or pages.

Here is what one of the template page looks like with the new theme. No code or styling was changed on this page, the new colors come from the global style

![theme](https://user-images.githubusercontent.com/19194912/99156721-41bdc000-2691-11eb-950a-2cdebf72a40c.PNG)

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

Checked the styling on all of the current pages, removed instances of p and h from application to ensure styling works everywhere.

florisdobber reviewed (commented):
Looks great to me, super helpful! Let Jack take a look to see if the React makes sense.

florisdobber reviewed (commented):
Perfect I was looking into doing this but couldn't figure it out

jackblanc commented on craco.config.js:
```diff
@@ -8,7 +8,31 @@ module.exports = {
+              '@black': '#000000',
+              '@white': '#FFFFFF',
+              '@body-background': '@white',
+              '@component-background': '@white',
```
Nice job using variables for this

jackblanc commented on src/components/greeting-container/GreetingContainer.tsx:
```diff
@@ -0,0 +1,26 @@
+}) => {
+  return (
+    <div className="info-container">
+      <Title style={{ color: '#000000' }}>{header}</Title>
```
Is there a way to do this using variables?

jackblanc commented on src/components/greeting-container/greeting-container.less:
```diff
@@ -0,0 +1,19 @@
+.info-container {
```
We want to start using styled components for things like this, but I don't think it's installed on this repo so we can do it later, after a merge from frontend-scaffold

jackblanc commented on src/containers/signup/signup.less:
```diff
@@ -6,6 +6,11 @@
   background: white;
   overflow: auto;
+
+  .ant-typography.footer {
```
We want to avoid overwriting AntD styles, why is this change required?

jackblanc reviewed (approved):
Love the progress with this. There are a few things we'll want to change to use https://styled-components.com/ in the future, but we have to merge frontend-scaffold first.

jackblanc reviewed (commented):
This looks pretty good, there's a few places where we want to use styled-components now. Sorry that the directions changed with that. Reach out on Slack with questions!

willmt80 commented:
Changed some places to use styled components, no longer override antd styles and removed inline magic color constants. Did not change the input container stuff because I believe that might be a separate ticket that involves reorganizing those components a bit more heavily.

jackblanc commented on src/containers/login/login.less:
```diff
@@ -4,19 +4,15 @@
   height: 100%;
   width: 100%;
-  background: white;
+  background: @white;
```
🔥

jackblanc reviewed (approved):
Looks great!
